README.md

```diff
@@ -1,10 +1,30 @@
-# interop-2023-mobile-testing
-Mobile Testing Investigation in Interop 2023
+# interop-mobile-testing
+Mobile Testing Investigation in Interop
 
-## Scope
+## 2024 Investigation
+
+### Scope
+
+Do the work so that by Interop 2025 the dashboard has a mobile view, and the mobile view is considered as reliable as desktop.
```

Comment:
```suggestion
Do the work so that by Interop 2025 the dashboard has a mobile view, with at least "experimental" versions of browsers, and the mobile view is considered as reliable as desktop.
```